.github/workflows/js-ci.yml

```diff
@@ -216,7 +213,6 @@ jobs:
       WORKSPACE: keycloak-admin-ui
     strategy:
       matrix:
-        container: [1, 2, 3, 4, 5]
         browser: [chrome, firefox]
```

Comment:
Currently there is nothing set up for Playwright to actually run the tests in Firefox for the nighties. This will have to be changed so we're also testing on Firefox.

Reply:
right this is something we need to add, let's first get this in and then we can add that

Reply:
Sure, can be handled as a follow up. Can you log an issue for it?